src/Components/Home/FAQ.jsx

```
@@ -0,0 +1,158 @@
+import React from 'react';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
```

ESLint

[error] 'Link' is defined but never used.

(no-unused-vars)